src/runtime_config/gov_root_ca_cert_mismatch.lua

```diff
@@ -0,0 +1,126 @@
+-- Copyright (c) Microsoft Corporation. All rights reserved.
+-- Licensed under the Apache 2.0 License.
+
+-- This file defines the default initial contents (ie, Lua scripts) of the governance scripts table.
+
+return {
+  pass = [[
+  tables, calls, votes = ...
+  -- interface definitions
+  PASSED = 1
+  PENDING = 0
+  REJECTED = -1
+  STATE_ACTIVE = "ACTIVE"
+
+  -- Root CA validation definitions
+  -- This MRSIGNER is intentially random and does not match the actual certificate.
+  -- See the governance_root_ca_cert_mismatch.py test.
+  EXPECTED_MRSIGNER = "abcd"
```

Comment:
Its a shame to duplicate this entire file for this change. An ugly but smaller solution would be to grep for this line and replace it in the original script. I think I'd prefer that, but willing to be overruled.

Reply:
Or maybe this is the reason we were looking for to put it in a KV table?

Reply:
I'm happy to put it in a kv table once they can be created dynamically. I don't think it makes sense to hard-code a certain scenario now because the validation logic will depend on the exact claim checks that are done. Here we only compare MRSIGNER but this will likely change.

Reply:
So that's a fair point, and I'm not even sure dynamic tables will be enough because the constitution (or at least a constitution) must be there from the first transaction, before anyone has the opportunity to create any tables.

I think the template + replace (%%%MRSIGNER%%% or so) looks attractive. If the code does the instantiation, that helps merge the test scripts too (each test instantiates it with a different value)?